pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -115,6 +121,13 @@ pub mod pallet {
 
 		/// A relayer was removed.
 		RelayerRemoved { relayer: DomainAddress },
+
+		/// An outbound message has been submitted.
+		OutboundMessageSubmitted {
+			sender: T::AccountId,
+			message: Vec<u8>,
```

Comment:
Why not use a type here? Easier to read in the explorer.

Reply:
Like `Bytes`?